skeleton/frontend/src/Frontend.hs

```diff
@@ -1,24 +1,30 @@
 {-# LANGUAGE DataKinds #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE TypeApplications #-}
+
 module Frontend where
 
 import qualified Data.Text as T
 import qualified Data.Text.Encoding as T
+
 import Obelisk.Frontend
 import Obelisk.Configs
 import Obelisk.Route
+import Obelisk.Generated.Static
+
 import Reflex.Dom.Core
 
 import Common.Api
 import Common.Route
-import Obelisk.Generated.Static
+
 
 frontend :: Frontend (R FrontendRoute)
 frontend = Frontend
-  { _frontend_head = el "title" $ text "Obelisk Minimal Example"
+  { _frontend_head = do
+      el "title" $ text "Obelisk Minimal Example"
+      elAttr "link" ("href" =: static @ "main.css" <> "type" =: "text/css" <> "rel" =: "stylesheet") blank
```

Comment:
The space between `@` and `"main.css"` is inconsistent with pattern below. I think we prefer no space so it's a bit more obvious that `@` is not an operator.

Reply:
Right, my bad!